### PR TITLE
Feature/ch679/resizing group user and group volume

### PR DESCRIPTION
**Pull request by kentwelcome**
Implement resize volume in admin notebook

### CODE REVIEW

**Comment by aar0nTw** [comment]
LGTM

File: .circleci/config.yml [hunk 1]
```diff
@@ -180,6 +180,9 @@ jobs:
                   docker tag ${CONTAINER_NAME}:${DOCKER_TAG} ${CONTAINER_NAME}:latest
                   docker push ${CONTAINER_NAME}:latest
             fi
+  deploy_manually:
```
Comment (hSATAC) [inline .circleci/config.yml]: If we're going to build branches other than master than I think we'll need to fix the "update the latest tag part" or at least remove that section.

Comment (hSATAC) [reply .circleci/config.yml]: Otherwise it would be weird because the `latest` tag could be way behind the main branch.

Comment (kentwelcome) [reply .circleci/config.yml]: Remove latest tag

**Pull request merged**